packages/core/src/lib/actions/env.ts

```diff
@@ -0,0 +1,54 @@
+import type { IncomingHttpHeaders } from "http"
+import type { AuthAction, AuthConfig } from "../../types"
+
+/**
+ * Set default env variables on the config object
+ **/
+export function setEnvDefaults(envObject: any, config: AuthConfig) {
+  try {
+    const url = process.env.AUTH_URL
```

Comment:
Cannot leave `process` in the code lib, it needs to be fully runtime agnostic

```suggestion
    const url = envObject.AUTH_URL
```